src/main/java/org/dita/dost/util/DITAOTCopy.java

```diff
@@ -89,7 +89,11 @@ public void execute() throws BuildException {
             throw new BuildException("Destination directory not defined");
         }
         if (!destDir.exists()) {
-            throw new BuildException("Destination directory " + destDir + " does not exists");
+            try {
+                destDir.mkdir();
```

Comment:
This should be `mkdirs()`, not `mkdir()`, as there may be missing directories in multiple levels. Also, this should not just catch `SecurityException`; it should also check the return value of `mkdirs()` and throw an `IOException` if it returns `false`.